cmd/buildah/common.go

```diff
@@ -99,7 +100,7 @@ func getDateAndDigestAndSize(image storage.Image, store storage.Store) (time.Tim
 	if err != nil {
 		return created, "", -1, err
 	}
-	img, err := storeRef.NewImage(nil)
+	img, err := storeRef.NewImage(context.TODO(), nil)
```

Comment:
Go ahead and add a local function to `common.go` that'll return a context for use in the binary, have it return `context.TODO()`, and use it instead.  It'll be a smaller change to not use TODO later.

Reply:
fixed. PTAL